lib/json/common.rb

```diff
@@ -152,7 +152,7 @@ class MissingUnicodeSupport < JSONError; end
   # * *object_class*: Defaults to Hash
   # * *array_class*: Defaults to Array
   def parse(source, opts = {})
-    Parser.new(source, opts).parse
+    Parser.new(source, **(opts||{})).parse
```

Comment:
In params we are already assign the empty hash to opts  `opts = {}` do we need to explictly do `|| {}` ?

Reply:
done